admin_manual/appliance/index.rst

```diff
@@ -15,3 +15,4 @@ The ownCloud X Appliance
    Active_Directory
    Backup
    Office
+   firewall
```

Comment:
Would be good to have all entries here in lowercase, as with the remainder of the documentation.

Reply:
It is already lowercase, isn't it?

Reply:
It is, but the three preceeing ones aren't.

Reply:
Something for the future.